tensorflow/docs_src/tutorials/image_retraining.md

```diff
@@ -115,7 +115,7 @@ process is progressing. The training's objective is to make the loss as small as
 possible, so you can tell if the learning is working by keeping an eye on
 whether the loss keeps trending downwards, ignoring the short-term noise.
 
-By default this script will run 4,000 training steps. Each step chooses ten
+By default this script will run 4,000 training steps. By default, each step chooses 100
```

Comment:
nit: Can you redo the line breaks to obey the 80-char line width limit?

Reply:
Will this work? I just realized the "by default" was repetitive anyways.